bootstrap/backends/ldap/ldap.go

```diff
@@ -91,7 +91,7 @@ func NewIdentityManager(bs bootstrap.Bootstrap) (identity.Manager, error) {
 
 	// Use a clone here to avoid changing the config of other possible users of the config.
 	tlsConfig := config.TLSClientConfig.Clone()
-	if caCertFile := os.Getenv("LDAP_TLS_CACERT"); caCertFile != "" {
+	if caCertFile := os.Getenv("LDAP_TLS_CACERT"); caCertFile != "" && !tlsConfig.InsecureSkipVerify {
```

Comment:
@rhafer what is the reason for this? Imo if a certificate is configured it should also be loaded, even if it is not used because of other configuration. Please add some more details what problem this actually solves. Right now i would prefer not to merge this.

Reply:
The main reason was, that the certificate (even if loaded) is not used for anything when  `InsecureSkipVerify` is set. So why load it at all. But I am fine with not merging this.